packages/gestalt/src/SheetMobile.js

```diff
@@ -104,6 +105,10 @@ type Props = {|
    * Sets the SheetMobile's height. See the [size variant](https://gestalt.pinterest.systems/web/sheetmobile#Size) for more info.
    */
   size?: 'default' | 'full' | 'auto',
+  /**
+   * An object representing the zIndex value of the Dropdown menu. Learn more about [zIndex classes](https://gestalt.pinterest.systems/web/zindex_classes)
```

Comment:
This should reference SheetMobile and not Dropdown, yes?

Reply:
DONE